e2e/tests/integration/arrivals.ts

```diff
@@ -160,7 +160,7 @@ describe('Arrivals page', () => {
           ukPassport
         ]
       ))
-      .get('.pax-api')
+      .get('.pax-unknown')
```

Comment:
Have we considered the case of ports that don't have live feeds, e.g. LPL, where we will never have a live feed total to verify the API total against? I think we probably don't have a way to make E2E tests for that at the moment.

Reply:
Unit test covers checks for LivePortFeed check .